### PR TITLE
Resolve CVS-82065

### PR DESCRIPTION
**Pull request by vinnamkim**
Related issues
 - https://jira.devtools.intel.com/browse/CVS-82065
 - https://jira.devtools.intel.com/browse/CVS-83115
 - https://jira.devtools.intel.com/browse/CVS-83116

### CODE REVIEW

File: external/mmdetection/tests/test_ote_api.py [hunk 1]
```diff
@@ -542,14 +542,13 @@ def test_training_yolox(self):
...
     @e2e_pytest_api
-    @pytest.mark.xfail(reason='CVS-83115')
     def test_training_maskrcnn_resnet50(self):
         self.end_to_end(osp.join('configs',
                         'custom-counting-instance-seg', 'resnet50_maskrcnn'),
                         task_type=TaskType.INSTANCE_SEGMENTATION)
 
     @e2e_pytest_api
-    @pytest.mark.xfail(reason='CVS-83116')
+    @pytest.mark.xfail(reason='It randomly cannot pass export_perf_delta_tolerance or pot_perf_delta_tolerance')
```
Comment (goodsong81) [inline external/mmdetection/tests/test_ote_api.py]: Please use Jira ticket ID. You may want to edit existing one or add another.

**Review by goodsong81 — CHANGES_REQUESTED** [review]

**Comment by goodsong81** [comment]
run ote-test

File: external/mmdetection/tests/test_ote_api.py (hunk 1 above)
Comment (vinnamkim) [reply external/mmdetection/tests/test_ote_api.py]: Fixed

**Comment by goodsong81** [comment]
Merging for early RC test on SC.

**Pull request merged**